Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/task_list/TaskListActivity.java

```diff
@@ -12,13 +13,31 @@
 import database.entities.ChildPlan;
 import database.entities.TaskTemplate;
 import database.repository.ChildPlanRepository;
+import pg.autyzm.friendly_plans.ActivityProperties;
 import pg.autyzm.friendly_plans.App;
 import pg.autyzm.friendly_plans.R;
+import pg.autyzm.friendly_plans.child_app.view.step_list.StepListActivity;
 
 public class TaskListActivity extends AppCompatActivity {
 
     @Inject
     ChildPlanRepository childPlanRepository;
+    private TaskRecyclerViewAdapter taskRecyclerViewAdapter;
+
+    TaskRecyclerViewAdapter.TaskItemClickListener taskItemClickListener =
```

Comment:
Not sure if it shouldn't be moved to other class. I would say, that it would be nice, to keep at least activities quite clean.